tests/functional/rbac/test_reassignment_data/provider/actions.yaml

```diff
@@ -0,0 +1,23 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: "SeeMeInAction"
+  hosts: localhost
+  tasks:
+    - name: "Hehe"
+      assert:
+        that:
+          - 1 == {{job.config.some_param}}
+        msg: "Never gonna give you up"
+      tags:
+        - compare
```

Comment:
don't think the action file is required, so I suggest deleting it

Reply:
removed